src/patternfly/components/Drawer/drawer.scss

```diff
@@ -278,6 +334,68 @@ $pf-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   min-height: 100%;
 }
 
+.pf-c-drawer__splitter {
+  position: absolute;
+  top: var(--pf-c-drawer__splitter--Top);
+  right: var(--pf-c-drawer__splitter--Right);
+  bottom: var(--pf-c-drawer__splitter--Bottom);
+  left: var(--pf-c-drawer__splitter--Left);
+  display: none;
+  width: var(--pf-c-drawer__splitter--Width);
+  height: var(--pf-c-drawer__splitter--Height);
+  cursor: var(--pf-c-drawer__splitter--Cursor);
+  visibility: visible;
```

Comment:
why is this display: none and visibility: visible ?

Reply:
because I oof'd 👍